.github/ISSUE_TEMPLATE/new-wolfi-package-request.yml

```diff
@@ -1,10 +1,14 @@
 name: New Wolfi Package Request
-description: Use this form to request the inclusion of wolfi OS. Please review our wolfi [package policy](wolfi.dev/os#if-wolfi-is-missing-a-package-your-require) before requesting a package.
+description: Use this form to request the inclusion of wolfi OS.
 title: "[Wolfi Package Request]: $PACKAGE_NAME"
 labels: [ "wolfi-package-request", "needs-triage" ]
 assignees:
-  - kanini
+  - kaniini
 body:
+  - type: markdown
+    attributes:
+      value: |
+        We appreciate your interest in Wolfi!  Please review our [description of appropriate packages and licenses](wolfi.dev/os#if-wolfi-is-missing-a-package-your-require) before submitting a request.
```

Comment:
```suggestion
        We appreciate your interest in Wolfi!  Please review our [description of appropriate packages and licenses](https://github.com/wolfi-dev/os#if-wolfi-is-missing-a-package-you-require) before submitting a request.
```

Reply:
I don't care much about the wolfi.dev vs github.com, but the # part is broken is all.

Reply:
:+1: thanks for the catch.